Python/emscripten_trampoline.c

```diff
@@ -0,0 +1,78 @@
+#if defined(PY_CALL_TRAMPOLINE)
+
+#include <emscripten.h>
+#include <Python.h>
+
+static int type_reflection_available;
```

Comment:
This should _probably_ go in the interpreter state (`PyInterpreterState`).

Reply:
well I think it is not necessary to put it there. I don't see anything in `PyInterpreterState` that is about runtime determination of the architecture. It is okay for this to be a mutable global because it is actually measuring a property of the "hardware" that the Python is running on. So if two interpreters are started and it is initialized in one or both, it will always get assigned the same value both times. As long as the value is filled in at least once, it's okay.

Though maybe it's harmless to put it there. I'm certainly happy either way =)

Reply:
You can put in the global runtime struct `_PyRuntime`, we have consolidated all globals there.

Reply:
Thanks for the info @kumaraditya303, will do.